### PR TITLE
Add option to use Mozilla CA bundle

### LINKED ISSUE #100: Use Mozilla CA bundle to validate peers
LibCURL 0.6 switched to using LibCURL_jll & MozillaCACerts_jll (https://github.com/JuliaWeb/LibCURL.jl/pull/75). 

`curl_easy_setopt(curl, CURLOPT_CAINFO, LibCURL.cacert)` must be added to this package to use the Mozilla CA bundle to validate peers (https://github.com/JuliaWeb/LibCURL.jl#ssl-certificates). The use of LibCURL in this package currently has no associated CA info which means all certs are being treated as self-signed.

### PR DESCRIPTION
**Pull request by mjram0s**
Closes #100

### CODE REVIEW

**Review by mattBrzezinski — APPROVED** [review]

**Comment by mjram0s** [comment]
The changes in this MR require LibCURL 0.6 which requires `julia = "1.3"` (https://github.com/JuliaWeb/LibCURL.jl/blob/87ca1d111cedb2573e0c170143b4b9002f751f20/Project.toml#L12). Not sure how to handle this but I'm guessing I need to: 
1. Drop support for LibCURL 0.5.1 (i.e. `LibCURL = "0.6"`)
2. Require `julia = "1.3"`
3. Tag as breaking (i.e. `version = "2.0.0"`)?

**Comment by iamed2** [comment]
I don't know that we need to do number 3. This may be okay as a minor change, given that there is no API change here.

**Comment by mattBrzezinski** [comment]
> I don't know that we need to do number 3. This may be okay as a minor change, given that there is no API change here.

If this is a minor version change, then this package would just not work with Julia pre-1.3?

**Comment by iamed2** [comment]
Pkg will ignore versions with Julia versions greater than the Julia that is running.

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ arch:
   - x64
   - x86
 julia:
-  - 1.0  # Latest LTS release
+  # - 1.0  # Latest LTS release (currently unsupported - see LibCURL 0.6)
   - 1    # Latest stable release
```
Comment (iamed2) [inline .travis.yml:12]:
```suggestion
  - 1.3  # Earliest supported release
  - 1    # Latest stable release
```

We should test on our earliest supported release

**Review by iamed2 — APPROVED** [review]

**Pull request merged**